.pre-commit-config.yaml

```diff
@@ -1,15 +1,15 @@
 repos:
 - repo: https://github.com/asottile/pyupgrade
-  rev: v2.31.0
+  rev: v2.31.1
   hooks:
     - id: pyupgrade
       args: [--py37-plus]
 - repo: https://github.com/4Catalyzer/fourmat
-  rev: v0.10.1
+  rev: v0.11.1
   hooks:
     - id: fourmat
 - repo: https://github.com/asottile/blacken-docs
   rev: v1.12.0
   hooks:
   - id: blacken-docs
-    additional_dependencies: [black==21.11b1]
+    additional_dependencies: [black==21.12b0]
```

Comment:
Can't update to latest due to https://github.com/asottile/blacken-docs/issues/141